docs/source/using_yosys/more_scripting/data_flow_tracking.rst

```diff
@@ -0,0 +1,71 @@
+Dataflow tracking
+-------------------
+Yosys can be used to answer questions such as "can this signal affect this other signal?" via its *dataflow tracking* support.
+For this, four special cells, ``$get_tag``, ``$set_tag``, ``$overwrite_tag`` and ``$original_tag`` are inserted into the design and then the ``dft_tag`` is run, which converts these cells into ordinary logic.
+Typically, one would then use ``sby`` to prove assertions involving these cells.
```

Comment:
I think this is the only place outside of the introduction that mentions SBY, so it would probably be good to have a link here.